Models/TaskContext.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace TimeMgmt.Models
+{
+    public class TaskContext : DbContext
+    {
+        public TaskContext(DbContextOptions<TaskContext> options) : base(options)
+        {
+            //leave blank for now
+        }
+
+        public DbSet<TaskContext> Tasks { get; set; }
+        public DbSet<Category> Categories { get; set; }
+        //public string Categories { get; set; }
+
+
+        protected override void OnModelCreating(ModelBuilder mb)
+        {
+
+            mb.Entity<Category>().HasData(
+                    new Category { categoryid = 1, categoryname = "Home" },
```

Comment:
Sorry this is mostly capitalization stuff, because when you change it, you'll have to change it everywhere. 
I would suggest clicking on the property name in the model file and hitting `F2`, which should update it everywhere.